[0mBringing machine 'default' up with 'virtualbox' provider...[0m
[1m==> default: Checking if box 'ubuntu/trusty64' is up to date...[0m
[1;33m==> default: A newer version of the box 'ubuntu/trusty64' is available! You currently
==> default: have version '20161214.0.0'. The latest is version '20170222.0.0'. Run
==> default: `vagrant box update` to update.[0m
[1m==> default: Machine already provisioned. Run `vagrant provision` or use the `--provision`
==> default: flag to force provisioning. Provisioners marked to run always will still run.[0m
[1m==> default: Running triggers after up...[0m
[1m==> default: Executing remote command "cd /home/vagrant/MockApplication; /usr/local/bin/ruby-2.3.0/bin/rails server -d -b 0.0.0.0 --pid /home/vagrant/mock.pid"...[0m
[1;31m==> default: The SSH connection was unexpectedly closed by the remote end. This
==> default: usually indicates that SSH within the guest machine was unable to
==> default: properly start up. Please boot the VM in GUI mode to check whether
==> default: it is booting properly.[0m
[1m==> default: Remote command execution finished.[0m
[0;31mThe remote command "cd /home/vagrant/MockApplication; /usr/local/bin/ruby-2.3.0/bin/rails server -d -b 0.0.0.0 --pid /home/vagrant/mock.pid" returned a failed exit
code or an exception. The error output is shown below:

The SSH connection was unexpectedly closed by the remote end. This
usually indicates that SSH within the guest machine was unable to
properly start up. Please boot the VM in GUI mode to check whether
it is booting properly.[0m
